android/src/main/java/com/reactnativefacetec/Processors/AuthenticateProcessor.java

```diff
@@ -0,0 +1,177 @@
+//
+// Welcome to the annotated FaceTec Device SDK core code for performing secure Authentication!
+//
+package com.reactnativefacetec.Processors;
+
+import android.content.Context;
+import android.os.Handler;
+import android.os.Looper;
+import android.util.Log;
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.MediaType;
+import okhttp3.RequestBody;
+import org.json.JSONException;
+import org.json.JSONObject;
+import java.io.IOException;
+
+//import com.facetec.sampleapp.SampleAppActivity;
+import com.facetec.sdk.*;
+
+// This is an example self-contained class to perform Authentication with the FaceTec SDK.
+// You may choose to further componentize parts of this in your own Apps based on your specific requirements.
+
+// Android Note 1:  Some commented "Parts" below are out of order so that they can match iOS and Browser source for this same file on those platforms.
+// Android Note 2:  Android does not have a onFaceTecSDKCompletelyDone function that you must implement like "Part 10" of iOS and Android Samples.  Instead, onActivityResult is used as the place in code you get control back from the FaceTec SDK.
+public class AuthenticateProcessor extends Processor implements FaceTecFaceScanProcessor {
+    private boolean isSuccess = false;
+//    private SampleAppActivity sampleAppActivity;
+
+    public AuthenticateProcessor(String sessionToken, Context context) {
```

Comment:
Same with this file - I'm not sure we need an AuthenticateProcessor. We really just need the LivenessProcessor at this time. That's what all of them are doing.